float32[] localwaypointsx
float32[] localwaypointsy

